lib/PuppeteerSharp/LaunchOptions.cs

```diff
@@ -11,19 +11,24 @@ namespace PuppeteerSharp
     /// </summary>
     public class LaunchOptions : IBrowserOptions, IConnectionOptions
     {
+        private string[] _ignoredDefaultArgs;
+
         /// <summary>
-        /// Whether to ignore HTTPS errors during navigation. Defaults to false.
+        /// Initializes a new instance of the <see cref="T:PuppeteerSharp.LaunchOptions"/> class.
         /// </summary>
-        public bool IgnoreHTTPSErrors { get; set; }
+        public LaunchOptions()
+        {
+            DefaultViewport = new ViewPortOptions
```

Comment:
this code appears also in the `ConnectOptions` ctor, maybe add a `Default` static readonly property on the `ViewPortOptions` class?